lib/board.rb

```diff
@@ -0,0 +1,64 @@
+class Board
+  attr_accessor :cells, :token
+  def initialize(amount=9,cell=" ")
+    @cells = []
+    @token = "X"
+    amount.times do |i|
+      @cells << cell
+    end
+  end
+
+  def reset!
+    @cells.clear
+    9.times do |i|
+      @cells << " "
+    end
+  end
+
+  def display
+    puts " #{cells[0]} | #{cells[1]} | #{cells[2]} "
+    puts "-----------"
+    puts " #{cells[3]} | #{cells[4]} | #{cells[5]} "
+    puts "-----------"
+    puts " #{cells[6]} | #{cells[7]} | #{cells[8]} "
+  end
+
+  def position(input)
+    return @cells[input.to_i - 1]
+  end
+
+  def full?
+    @cells.include?(" ") ? (return false) : (return true)
```

Comment:
Use the most expressive and efficient iterator for a problem. `Board#full?` is asking if all elements are not empty strings. You can use logic that is symmetrical to those semantics with the [`#all?`](http://ruby-doc.org/core-2.2.3/Enumerable.html#method-i-all-3F) iterator.

`cells.all? {|cell| cell != " "}`

This is also more efficient in that the loop will stop after the iteration for which a cell is not empty.